p2p/src/node.rs

```diff
@@ -1256,7 +1256,7 @@ mod tests {
         type Key = K;
         type Hash = Sha256a;
 
-        async fn insert(&self, _items: Vec<ReconItem<Self::Key>>) -> ReconResult<()> {
+        async fn insert(&self, _items: Vec<ReconItem<Self::Key>>) -> ReconResult<InsertBatch<K>> {
```

Comment:
why `InsertBatch<K>` instead of `InsertBatch<Self::Key>`?  They're aliases right?  Just seems weird to use `Key` in the arguments but `K` in the return type, seems better to be consistent?

Reply:
oversight :).. I just typed what I was typing in other places and will change it